### PR TITLE
4 2 tests pass

### PR DESCRIPTION
**Pull request by glongman**
might be confusing.

Yesterday I made this PR to get tests running in 4.1 + r2.3.3 https://github.com/HealthWave/spree/pull/21

It remains unmerged until a test in hw_admin (r2.1.6) is complete.
https://github.com/HealthWave/spree/pull/22/commits/ed5db91768ba89e077a728c513cdd782ad4d1b87
https://github.com/HealthWave/spree/pull/22/commits/0b7fd4bbd27323b8b1c45c00f80cce13acf5e6f3

last commit is the meat of getting the tests to pass in 4.2 + r2.3.3
https://github.com/HealthWave/spree/pull/22/commits/a4cd0ac9b84f42cd3152615664c503b16d89ed99

### CODE REVIEW

**Comment by glongman** [comment]
do not merge. I think the target branch for the merge is incorrect. checking...

**Comment by glongman** [comment]
disregard my last - can be merged when reviewed.

File: core/lib/spree/core/mail_method.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@ class MailMethod
       def initialize(options={})
       end
 
-      def deliver!(mail)
+      def deliver_now!(mail)
         if Config.enable_mail_delivery
-          mailer.deliver_now!(mail)
+          mailer.deliver!(mail)
```
Comment (kurtfunai) [inline core/lib/spree/core/mail_method.rb:9]: I think this should still be able to use `deliver_now!` as it is the method in 4-2

**Review by kurtfunai — COMMENTED** [review]
Well documented changes and makes a lot of sense. Only had one question on the PR .

File: core/lib/spree/core/mail_method.rb (hunk 1 above)
Comment (glongman) [reply core/lib/spree/core/mail_method.rb:9]: I futz'd with this quite a bit. It's a spree method call not ActionMailer. Goal was not to change spree call signatures.

**Comment by glongman** [comment]
question answered

File: core/lib/spree/core/mail_method.rb (hunk 1 above)
Comment (kurtfunai) [reply core/lib/spree/core/mail_method.rb:9]: makes sense

**Comment by kurtfunai** [comment]
I think this lgtm

**Pull request merged**